Extension/package.json

```diff
@@ -42,6 +42,168 @@
+          {
+            "id": "build.project.linux",
+            "title": "Build your C++ file",
+            "description": "Create a default build task for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-linux#_build-helloworldcpp)",
+            "when": "isLinux",
+            "media": {
+              "altText": "markdown file with instructions for configuring default build task on Linux with GCC",
+              "path": "walkthrough/build-project-linux.md"
+            }
+          },
+          {
+            "id": "build.project.windows",
+            "title": "Build your C++ file",
+            "description": "Create a default build task for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-msvc#_build-helloworldcpp)",
+            "when": "isWindows",
+            "media": {
+              "altText": "markdown file with instructions for configuring default build task on MSVC with Windows",
+              "path": "walkthrough/build-project-windows.md"
+            }
+          },
+          {
+            "id": "run.project.mac",
+            "title": "Run and debug your C++ file",
+            "description": "Create a debug configuration for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-clang-mac#_debug-helloworldcpp)",
+            "when": "isMac",
+            "media": {
+              "altText": "markdown file with instructions for starting a debugging session on macOS",
+              "path": "walkthrough/run-and-debug-project-mac.md"
+            }
+          },
+          {
+            "id": "run.project.linux",
+            "title": "Run and debug your C++ file",
+            "description": "Create a debug configuration for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-linux#_debug-helloworldcpp)",
+            "when": "isLinux",
+            "media": {
+              "altText": "markdown file with instructions for starting a debugging session on Linux",
+              "path": "walkthrough/run-and-debug-project-linux.md"
+            }
+          },
+          {
+            "id": "run.project.windows",
+            "title": "Run and debug your C++ file",
+            "description": "Create a debug configuration for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-msvc#_debug-helloworldcpp)",
+            "when": "isWindows",
+            "media": {
+              "altText": "markdown file with instructions for starting a debugging session on Windows",
+              "path": "walkthrough/run-and-debug-project-windows.md"
+            }
+          },
+          {
+            "id": "install.cmake.tools",
+            "title": "Install CMake Tools",
+            "description": "Do you build your project with CMake? Install the CMake Tools extension to seamlessly build and debug CMake project. \n[Install CMake Tools](command:workbench.extensions.installExtension?%22ms-vscode.cmake-tools%22)",
```

Comment:
Should we add a reference to Make and the Makefile Tools extension too?

Reply:
In the future, yes! We need a way to detect presence of a Makefile in a workspace.